addon/model-data.js

```diff
@@ -183,6 +183,10 @@ export default class M3ModelData {
 
   removeFromHasMany() {}
 
+  _initRecordCreateOptions(options) {
```

Comment:
Why is this method starting with an underscore? It seems part of the public Record Data API as it is not internal to ember-m3.

Reply:
because we need to RFC it in ember-data and it's going to have to evolve a bit still